doozer/doozerlib/cli/scan_fips.py

```diff
@@ -26,12 +28,21 @@ def __init__(self, runtime: Runtime, nvrs: Optional[list], clean: Optional[bool]
         self.runtime.initialize(clone_distgits=False)
         self.koji_session = koji.ClientSession(self.runtime.group_config.urls.brewhub)
 
+        if self.nvrs and self.all_images:
+            raise Exception("Cannot specify both --nvrs and --all-images")
+
     async def execute_and_handle_cmd(self, cmd, nvr):
         rc, out, _ = cmd_gather(cmd)
         if rc != 0:
             self.could_not_clean.append(nvr)
         return rc, out
 
+    @staticmethod
+    async def clean_all_images():
+        rc, _, _ = cmd_gather("podman image prune --all --force")
+        if rc != 0:
+            raise Exception("Could not clean all images")
```

Comment:
If `cmd_assert_*` is used, there will be no need to check the return value.
Also this exception message doesn't include the original error message from the executed command, which could be hard to debug in case something goes wrong.